### PR TITLE
Example convex_decomposition

### PR DESCRIPTION
**Pull request by Vrixyz**
- Bunny can take ~25 seconds to decompose on debug (on a good cpu) ; I added a warning against running on debug.

image

- stairs is quicker to load, but arguably less visually appealing:

<details><summary>Details</summary>
<p>

image

</p>
</details>

### CODE REVIEW

**Comment by Vrixyz** [comment]
- We should arbitrate against it to be used for investigating current issues, and make a different PR to investigate https://github.com/dimforge/rapier/issues/223

File: crates/parry3d/examples/convex_decomposition.rs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            .polys
+            .iter()
+            .map(|p| [p.0[0].0 as u32, p.0[1].0 as u32, p.0[2].0 as u32])
+            .collect::<Vec<_>>(),
+        TriMeshFlags::all(),
+    )
+    .unwrap();
+    clear_background(BLACK);
+
+    easy_draw_text("Please wait while convex decomposition is being computed...");
```
Comment (Vrixyz) [inline crates/parry3d/examples/convex_decomposition.rs:41]: TODO: if debug mode, add another sentence to advice to run it in release

File: crates/parry3d/examples/convex_decomposition.rs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    #[cfg(debug_assertions)]
+    {
+        macroquad::text::draw_text(
+            "Running in debug mode is way slower than with `--release`.",
```
Comment (sebcrozet) [inline crates/parry3d/examples/convex_decomposition.rs:45]:
```suggestion
            "Running in debug mode is significantly slower than with `--release`.",
```

File: crates/parry3d/examples/convex_decomposition.rs [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        set_camera(&Camera3D {
+            position: camera_pos,
+            up: Vec3::new(0f32, 1f32, 0f32),
+            target: Vec3::new(0f32, 1f32, 0f32),
+            ..Default::default()
+        });
+
+        let shapes_count = trimesh_convex_compound.shapes().len() as u32;
+        for (i, s) in trimesh_convex_compound.shapes().iter().enumerate() {
+            let trimesh_convex = s.1.as_convex_polyhedron().unwrap().to_trimesh();
```
Comment (sebcrozet) [inline crates/parry3d/examples/convex_decomposition.rs:76]: Looks like this conversion could be done only once outside of the loop.

**Review by sebcrozet — APPROVED** [review]
Thanks!

**Pull request merged**